src/framework/theme/components/user/user.component.html

```diff
@@ -9,7 +9,7 @@
     <nb-badge *ngIf="badgeText" [text]="badgeText" [status]="badgeStatus" [position]="badgePosition"></nb-badge>
   </div>
 
-  <div class="info-container">
+  <div class="info-container" *ngIf="!onlyPictureValue">
```

Comment:
Do we really need to introduce new variable here? Maybe, we'll just add this if?
```
  <div class="info-container" *ngIf="showNameValue || showTitleValue">
```

Reply:
@lexzhukov, What do you think?

Reply:
I've thought for your example. I don't like "mode" behavior without a special flag, but we can use property like `onlyPictureMode = showNameValue || showTitleValue`.

Reply:
I see it 😄 
But could you please explain why introducing new variable is better than using existing?

Reply:
It's explicit behavior in one word. I don't want to spend time on understanding multiple variables in one row:)

Reply:
Ok, It's clear enough, so, let's create a getter instead